### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**
Hi, here is my JS1-week2 homework.
I got a lot of extra help to finish it.

### CODE REVIEW

**Review by Impressiv3 — CHANGES_REQUESTED** [review]

**Comment by RodrigoGuinsbergPinto** [comment]
Hi Bálint,
I have made the changes you suggested to the excess of comments, strict ===, switch statements and finished the candy function.
Thank you for the feedback.

File: javascript/javascript1/week2/js-week2.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+console.log(boughtCandyPrices);
+
+
+
+const amountToSpend = Math.floor(Math.random() * 100);
+// I can't get this to work.
+function canBuyMoreCandy() {
+    let total = 0;
+    for(let i = 0; i < boughtCandyPrices.length; i++) {
+    if (total <= amountToSpend) {
```
[the remark this answers is not in the document]
Comment (RodrigoGuinsbergPinto) [reply javascript/javascript1/week2/js-week2.js]: Thanks. I fixed it.

File: javascript/javascript1/week2/js-week2.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    console.log(`You can buy more, so please do!`);
+  } else {
+    console.log(`Enough candy for you!`);
+  }
+}
+}
+
```
[the remark this answers is not in the document]
Comment (RodrigoGuinsbergPinto) [reply javascript/javascript1/week2/js-week2.js:149]: Yes, I sure do. : )
Thanks.

**Comment by Impressiv3** [comment]
Very well done!  Good job !

**Review by Impressiv3 — APPROVED** [review]
I can see you already have merged it, but we still got some problems.
I will  contact you on Slack and we are going to fix them ;)

**Pull request merged**